static/css/gridpage.css

```
@@ -17,12 +17,18 @@
 
 .launch-card {
     position: relative;
-    width: 33%;
-    padding: 0 30px 0 0;
-    margin-bottom: 50px;
+    margin-bottom: 30px;
     min-height: 152px;
 }
 
+@media (min-width: 768px){
```

Looking at @kbhawkey's feedback, I suggest:
```suggestion
@media screen and (max-width: 768px){
```
?